google-cloud-spanner/src/main/java/com/google/cloud/spanner/GrpcStruct.java

```diff
@@ -94,6 +94,9 @@ private Object writeReplace() {
         case PG_NUMERIC:
           builder.set(fieldName).to((String) value);
           break;
+        case UUID:
+          builder.set(fieldName).to((String) value);
```

Comment:
I think this `value` is from backend as string. Should we be using `fromString` to convert string value to java.util.UUID value? I don't think this is the right place to do this conversion. May be when customer reads it as UUID then we should use the fromString method.
https://docs.oracle.com/javase/8/docs/api/java/util/UUID.html